### ISSUE TITLE
Add Saitek Yoke support

### ISSUE THREAD

**Issue by mdjarv** [mdjarv/JoystickVisualizer#21]
As mentioned in #17, model and implement support for the Saitek Yoke

**Comment by brittz** [comment]
Hi, I've tested with Saitek Yoke, but nothing shows up on the Visualizer. I can see the Flight Yoke device in the Proxy. (version JoystickVisualizer-0.11.8)

**Comment by mdjarv** [comment]
Yes, this ticket is about adding the yoke support, meaning it has not yet been implemented, I've done a few attempts at modeling the yoke but haven't been happy with the results.

It is in the feature pipeline and this issue will be updated once I've made some progress
